code/__HELPERS/icons.dm

```diff
@@ -915,6 +919,12 @@ var/global/list/friendly_animal_types = list()
 		final_average = BlendRGB(final_average, colour, 1)
 	return final_average
 
+/proc/IsBlankIcon(icon/I)

```

Comment:
Oh don't worry about it, due to cruel fate ```/icon```'s have no appearance, but guess what? the principle behind what i said works perfectly, except we just can't use it because ```/icon``s are poo.